packages/experimental/Divider/src/Divider.styling.ts

```diff
@@ -0,0 +1,104 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { layoutStyles } from '@fluentui-react-native/tokens';
+import { DividerTokens, DividerSlotProps, DividerAppearance } from './Divider.types';
+import { IconPropsV1 } from '@fluentui-react-native/icon';
+
+export const getDividerSlotProps = (tokens: DividerTokens, theme: Theme, icon?: IconPropsV1): DividerSlotProps => {
+  let iconProps: IconPropsV1 = {};
```

Comment:
Create a new function getIconProps to that returns iconProps instead of putting this code bit in this function.